docker/video/haproxy.conf

```diff
@@ -24,31 +27,34 @@ global
     #no option       httpclose
     #no option       http-server-close
     #timeout         client 7200s
-    option http-keep-alive
+    #option http-keep-alive
+    mode http
     default_backend be_isard-squid
        
   frontend  fe_secured
     bind            0.0.0.0:443 ssl crt /certs/chain.pem
+    mode            http
+    log             global
+    option          httplog
+    timeout         client   3600s
+    backlog         4096
+    maxconn         50000      
+    option          httpclose
+    option          tcpka
 
     acl is_upgrade hdr(Connection) -i upgrade
     acl is_websocket hdr(Upgrade) -i websocket    
     use_backend be_isard-websockify if is_websocket
 
   backend be_isard-websockify
     option forwardfor
-    server websockify isard-websockify:8080
+    server websockify isard-websockify:8080 check port 8080 inter 5s rise 2 fall 3 resolvers mydns init-addr none
 
   backend be_isard-squid
-    option http-keep-alive
     option forwardfor
     option redispatch
     option abortonclose
-    server squid isard-squid:8080
-
- backend be_isard-hypervisor
```

Comment:
Did you delete be_isard-hypervisor on purpose?

Reply:
Yes, it is not used as backend (no redirects from any frontend). It should have been previous tests directly to hypervisor.